src/agentscope/agents/user_agent.py

```diff
@@ -101,10 +119,3 @@ def reply(
             self.memory.add(msg)
 
         return msg
-
-    def speak(
```

Comment:
when running as_gradio， the user input will be printed in both left and right hands (since `speak` will print the msg in the right hand)